docs/design_decisions.md

```diff
@@ -43,9 +43,9 @@ De identificatie van BAG-objecten wordt geïmplementeerd als string, waarin de d
 In het RSGB is de identificatie van BAG-objecten (bijvoorbeeld identificatiecode van een nummeraanduiding) opgenomen als complex datatype. In de BAG-registratie is dit echter één string waarin de deel-elementen uit het complex datatype zijn samengevoegd.
 Zie issue [#1](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/issues/1).
 
-## Eén uniform endpoint voor zoeken ingeschreven persoonsgegevens
-Voor ingeschreven personen komt er één endpoint voor het zoeken: /ingeschrevenNatuurlijkPersonen.
-Dit endpoint geeft alle attributen van de ingeschreven persoon van het LO GBA, geen aanhangende gegevens of gemeentelijke kerngegevens, alle relaties (die in GBA zitten) als link (uri).
+## Eén uniform endpoint voor zoeken persoonsgegevens
+Voor personen komt er één endpoint voor het zoeken: /ingeschrevenNatuurlijkPersonen.
```

Comment:
moet /ingeschrevenNatuurlijkPersonen ook niet worden aangepast?

Reply:
Zeker, die heb ik gemist. 
Die hebben we al eerder gemist want de endpoint ingeschrevenNatuurlijkPersonen kennen we al heel lang niet meer.